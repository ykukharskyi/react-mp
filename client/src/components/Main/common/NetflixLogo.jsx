import React from 'react';

import './NetflixLogo.scss';

const NetflixLogo = () => (
  <span className="netflix-logo">
    netflixroulette
  </span>
);

export default NetflixLogo;
