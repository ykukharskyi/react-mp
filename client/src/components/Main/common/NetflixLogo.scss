.netflix-logo {
  color: #e50914;
  font-weight: bold;
}
