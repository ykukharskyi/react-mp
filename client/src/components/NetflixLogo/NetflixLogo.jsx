import React from 'react';

import Span from './NetflixLogo.styles';

const NetflixLogo = () => (
  <Span className="netflix-logo">
    netflixroulette
  </Span>
);

export default NetflixLogo;
