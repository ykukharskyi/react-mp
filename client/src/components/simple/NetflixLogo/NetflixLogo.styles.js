import styled from 'styled-components';

export default styled.span`
  color: #e50914;
  font-weight: bold;
`;
