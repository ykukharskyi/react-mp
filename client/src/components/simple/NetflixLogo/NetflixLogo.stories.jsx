import React from 'react';
import { storiesOf } from '@storybook/react';
import NetflixLogo from './NetflixLogo';

storiesOf('NetflixLogo', module)
  .add('default logo', () => <NetflixLogo />);
