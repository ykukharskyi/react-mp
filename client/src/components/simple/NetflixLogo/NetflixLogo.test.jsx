import React from 'react';
import { shallow } from 'enzyme';
import NetflixLogo from './NetflixLogo';

describe('NetflixLogo', () => {
  it('default render', () => {
    const wrapper = shallow(<NetflixLogo />);
    expect(wrapper).toMatchSnapshot();
  });
});
