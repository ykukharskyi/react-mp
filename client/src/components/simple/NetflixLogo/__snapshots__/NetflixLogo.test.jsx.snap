// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetflixLogo default render 1`] = `
<styled.span
  className="netflix-logo"
>
  netflixroulette
</styled.span>
`;
